src/features/pages/options/digital/onlyups-onlydowns/data.ts

```diff
@@ -0,0 +1,143 @@
+import OnlyUpsDownsIcon from 'images/svg/trade-types/options/digital/only-ups-downs-icon.svg'
+import OnlyUpsWin from 'images/svg/trade-types/options/digital/only-ups-win.svg'
+import OnlyUpsLoss from 'images/svg/trade-types/options/digital/only-ups-loss.svg'
+import OnlyDownsWin from 'images/svg/trade-types/options/digital/only-downs-win.svg'
+import OnlyDownsLoss from 'images/svg/trade-types/options/digital/only-down-loss.svg'
```

Comment:
Can we add an `index` file on `digital` directory so we can import these on a minimal way?

Reply:
can. but I think we better add the `index` file once all the PRs of options contents got merged since many developers are workings on the different options page contents and adding images in the same directory.

Reply:
Okay I will create an initiative to clean up this after the release.